js/apps/system/_admin/aardvark/APP/react/src/views/g6graphs/ButtonSave.js

```diff
@@ -0,0 +1,67 @@
+/* global arangoHelper, $ */
+
+import React, { useContext, useState } from "react";
+import { UrlParametersContext } from "./url-parameters-context";
+import { Button, Tooltip } from 'antd';
+import { SaveOutlined } from '@ant-design/icons';
+
+const ButtonSave = ({ graphName, onGraphDataLoaded, onIsLoadingData }) => {
+  const urlParameters = useContext(UrlParametersContext);
+  const [isLoadingData, setIsLoadingData] = useState(false)
+  let responseTimesObject = {
+    fetchStarted: null,
+    fetchFinished: null,
+    fetchDuration: null
+  }
+  const [responseTimes, setResponseTimes] = useState(responseTimesObject);
+
+  const callApi = () => {
+    localStorage.setItem(`${graphName}-gv-urlparameters`, JSON.stringify(urlParameters[0]));
+    setIsLoadingData(true);
+    onIsLoadingData(true);
+    responseTimesObject.fetchStarted = new Date();
+
+    $.ajax({
```

Comment:
we have multiple methods to call APIs. `fetch` is used in some places and jquery in others. I'd suggest we avoid using jquery altogether in react code

Reply:
Will be taken care of in this ticket: https://arangodb.atlassian.net/browse/FE-203

Reply:
We will take this up later as jquery is used in a lot of places! Let's keep it in the backlog for now

Reply:
I agree. We can take this up later. Since it's a low-priority task so we should move this to backlog as Palash mentioned.